convert2rhel/unit_tests/main_test.py

```diff
@@ -113,6 +113,10 @@ def __call__(self, command, args):
             self.args = args
             return self.return_string, self.return_code
 
+    class GetCertMocked(unit_tests.MockFunction):
+        def __call__(self):
+            return "anything", "anything"
+
```

Comment:
Nit: you can use just Mock(return_value=("anything", "anything"), so you'll get what you need + bonus functionality

Reply:
@ZhukovGreen My motivation to do it as I did was to not combine two methods of testing in one test case. Or it doesn't matter?

Reply:
if yuo mean using mock.Mock in unittest, that's totally fine, moreover there is no mixing, as mock.Mock I snot related to pytest method (if you thought so)

Reply:
Other option: https://github.com/oamg/convert2rhel/pull/203#discussion_r657142772